staking/README.md

```diff
@@ -0,0 +1,541 @@
+# Staking Module
+
+## Table of Contents
+
+* [Preliminary terminology](#preliminary-terminology)
+* [Overview](#overview)
+* [How does one join the network as a validator](#how-does-one-join-the-network-as-a-validator)
+* [How to propose a MsgValidatorJoin transaction](#how-to-propose-a-msgvalidatorjoin-transaction)
+* [How does an existing validator exit the network](#how-does-an-existing-validator-exit-the-network)
+* [How to propose a MsgValidatorExit transaction](#how-to-propose-a-msgvalidatorexit-transaction)
+* [How does a validator update its stake](#how-does-a-validator-update-its-stake)
+* [How to propose a MsgStakeUpdate transaction](#how-to-propose-a-msgstakeupdate-transaction)
+* [How does a validator update its signer address](#how-does-a-validator-update-its-signer-address)
+* [How to propose a MsgSignerUpdate transaction](#how-to-propose-a-msgsignerupdate-transaction)
+* [Query commands](#query-commands)
+
+## Preliminary terminology
+
+* An `epoch` represents the period till a checkpoint is submitted on Ethereum, i.e one `epoch` ends when a checkpoint is committed on Ethereum and the next one begins.
```

Comment:
Not sure, but I'd probably say "An `epoch` represents the period in which a checkpoint is being submitted on Ethereum (i.e. one `epoch` ends when the checkpoint is committed on and the next one begins.)"

Reply:
Not sure whether you're pointing to the wording or suggesting the latter part be added in brackets or both but how about `An epoch represents the period until a checkpoint is submitted on Ethereum (i.e. one `epoch` ends when a checkpoint is committed on Ethereum and the next one begins).` ?

Reply:
See [159a33b](https://github.com/maticnetwork/heimdall/pull/1082/commits/159a33bafa2f43c7cc2ae0f9d57b4257ef036436)

Reply:
Yeah like that more, thanks.